src/components/SwitchViewButton/SwitchViewButton.tsx

```diff
@@ -0,0 +1,16 @@
+import { Button } from "@radix-ui/themes";
+import Gridview from "/src/assets/gridview.svg";
+import Listview from "/src/assets/listview.svg";
+import Image from "next/image";
+export default function SwitchViewButton() {
+  return (
+    <div className="flex items-center">
+      <Button variant="outline" color="gray" className="rounded-l-lg">
+        <Image src={Gridview} width={25} height={25} alt="logo" />
```

Comment:
Loading icon in image can be beneficial in some points but there is one downside that we cannot easily manipulate it styling via CSS or JavaScript. For example, hover to change icon color and it normally makes sense doing so inside a button. In the case, consider important the icon as a React component instead.